test/verify/check-login

```diff
@@ -343,6 +343,28 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         b.wait_not_visible("#login")
         b.wait_not_visible("#login-details")
 
+    @skipImage("Added in PR #12493", "rhel-8-0-distropkg")
+    def testFailingWebsocket(self):
+        m = self.machine
+        b = self.browser
+
+        # Cause cockpit-ws to reject WebSocket connections
+        m.write("/etc/cockpit/cockpit.conf", "[WebService]\nOrigins=foo.bar.com\n")
+        self.allow_journal_messages("received request from bad Origin: .*",
+                                    "Received invalid handshake request from the client")
+        self.allow_browser_errors(".*")
+
+        # Log in.
+        m.start_cockpit()
+        b.open("/system")
+        b.wait_visible("#login")
+        b.set_val('#login-user-input', "admin")
+        b.set_val('#login-password-input', "foobar")
+        b.click('#login-button')
+        b.expect_load()
+        b.wait_present("#content")
+
+        b.wait_visible("#early-failure")
```

Comment:
This should at least check that the safari bit isn't visible. But this could [set the user agent string](https://chromedevtools.github.io/devtools-protocol/1-3/Emulation#method-setUserAgentOverride) and then verify that this part works as well (to spot the bug above)

Reply:
Done.